docs/source/guide/zne-5-theory.myst

```diff
@@ -0,0 +1,110 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+# What is the theory behind ZNE?
+
+
+Zero noise extrapolation (ZNE) {cite}`Giurgica_Tiron_2020_arXiv` is an error
+mitigation technique used to extrapolate the noiseless expectation value of an
+observable from a range of expectation values computed at different noise levels.
+In Mitiq, this process works in two steps:
+
+- **Step 1: Intentionally scale noise** by manually inserting gates in a manner
+that keeps the logic of input circuit unchanged. The number of gates inserted is
+dependent on the scale factor. Different scale factor values will calculate the
+expectation value at different noise leveles.
+
+- **Step 2: Extrapolate to the noiseless limit** by fitting a curve [via one of
+the various methods available in Mitiq](https://mitiq.readthedocs.io/en/stable/apidoc.html#module-mitiq.zne.inference)
+to the expectation values calculated at different noise levels to extrapolate the
+noiseless expectation value.
+
+## Scaling Noise by Unitary Folding
+
+In ZNE, if a quantum circuit is implemented via gates, the circuit depth is increased
+for gates $G$ in the circuit by mapping as $G \mapsto G G^\dagger G$. Here, $G$
+can be a set of all the gates or a portion of gates in a quantum circuit. When all
+the gates are to be mapped as discussed previously, the process is known as *global
+folding*, while if only some gates in the circuit are to be folded, the procedure implemented
+is known as *local folding*. Thus, in either case noise caused due to physical implementations
+of unitary gates on gate-model devices is scaled by inserting additional gates
+in the circuit.
```

Comment:
```suggestion
A technique to increase the noise level of a circuit is to intentionally increase its depth.
This can be obtained using the *unitary folding* mapping $G \mapsto G G^\dagger G$. 
This mapping can be applied *globally* or *locally* as shown in the diagrams below.
```
This discussion is a bit complex to follow and could be simplified by simply referring to the diagrams.